htdocs/core/class/html.form.class.php

```diff
@@ -5305,10 +5305,14 @@ public function selectDate($set_time = '', $prefix = 're', $h = 0, $m = 0, $empt
 							// Note: We don't need monthNames, monthNamesShort, dayNames, dayNamesShort, dayNamesMin, they are set globally on datepicker component in lib_head.js.php
 							if (empty($conf->global->MAIN_POPUP_CALENDAR_ON_FOCUS))
 							{
-							$retstring.="
-								showOn: 'button',
-								buttonImage: '".DOL_URL_ROOT."/theme/".$conf->theme."/img/object_calendarday.png',
-								buttonImageOnly: true";
+								$imgpath = dol_buildpath($conf->theme."/theme/".$conf->theme."/img/object_calendarday.png");
```

Comment:
Using dol_buildpath each time we need to show an image or change a CSS lead to slowly dramatically pages generation. 
Being able to have an external module to change the look of a button must be addressed with CSS. So you should find a way to add a css or id on the object so external module can overload it ith CSS.

Reply:
you are totally right and i am ok with it.
since this commit, i thought a little more and remove the theme from my module; a simple css (module_parts) is much easier !